ant-replacement.gradle

```diff
@@ -220,12 +224,12 @@ void defineDeployTask(Object project, boolean debug) {
     task.doLast {
         ssh.run {
             session(remotes.robotAdmin) {
-                println "Copying natives"
-                put from: natives, into: nativesTarget
+                //println "Copying natives"
```

Comment:
should we get rid of these?

Reply:
no because we might need them very quickly and I wouldn't want to spend time looking them back up in commit logs